Vehicle/Vehicle.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AutonomousCars
+{
+    public class Vehicle : IVehicle
```

Comment:
Why separate Vehicle from VehicleInformation ? To me, having [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)) in mind, it would make more sense not to split, but am interested in your thoughts.